src/components/skillformular.tsx

```diff
@@ -1,5 +1,7 @@
 import React, { useState } from "react";
 import { Link } from "react-router-dom";
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
diese import Zeile müssten wir in jeder einzelnen .tsx Datei machen, was nicht vertretbar ist.
Wir müssen das bootstrap css iwie anders / woanders importieren.
Außerdem fehlt das bootstrap.min.js